### ISSUE TITLE
virtual_machine_scale_set documentation error 

### ISSUE THREAD

**Issue by hashibot** [terraform-providers/terraform-provider-azurerm#530]
_This issue was originally opened by @francisco-andrade as hashicorp/terraform#16569. It was migrated here as a result of the [provider split](https://www.hashicorp.com/blog/upcoming-provider-changes-in-terraform-0-10/). The original body of the issue is below._

<hr>

Hi there,

I found an error on the **[azurerm_virtual_machine_scale_set](https://www.terraform.io/docs/providers/azurerm/r/virtual_machine_scale_set.htm)** page at the setion:  **[Example of storage_profile_image_reference with id](https://www.terraform.io/docs/providers/azurerm/r/virtual_machine_scale_set.html#example-of-storage_profile_image_reference-with-id)**

When we plan a config using the example Terraform outputs an error:

> Error: azurerm_virtual_machine_scale_set.test: : invalid or unknown key: storage_image_reference

The problem is that the code is using **storage_image_reference** instead of **storage_profile_image_reference**.

**Wrong**
```
resource "azurerm_image" "test" {
    name = "test"
  ...
}

resource "azurerm_virtual_machine_scale_set" "test" {
    name = "test"
  ...

    storage_image_reference {
        id = "${azurerm_image.test.id}"
    }
...
```

**Right**
```
resource "azurerm_image" "test" {
    name = "test"
  ...
}

resource "azurerm_virtual_machine_scale_set" "test" {
    name = "test"
  ...

    storage_profile_image_reference {
        id = "${azurerm_image.test.id}"
    }
...
```

I'm running Terraform version 0.10.8.

Thanks,
Francisco Andrade

**Comment by VaijanathB** [comment]
Hello @francisco-andrade, 

Thanks for reporting this. I have created a PR for resolving this issue.  

Thanks, 
VaijanathB

**Comment by tombuildsstuff** [comment]
Fixed via #531
